### PR TITLE
Listing traditional tests

### PR DESCRIPTION
**Pull request by castelao**

### CODE REVIEW

**Review by soerenthomsen — DISMISSED** [review]

**Review by catsch — DISMISSED** [review]

File: src/tests.adoc [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+X_i = V_i - V_{i-1}
+++++
+
+== Spike - Median
+
+Proposed at <<BGC_nitrate>>
+
+[stem]
+++++
+X_i = \left| V_2 - median(V_0, V_1, V_2, V_3, V_4, V_5) \right|
```
Comment (callumrollo) [inline src/tests.adoc:32]: This notation is inconsistent with the other spike test above. This should be something like:

X_i = \left| V_i - median(V_i-2, V_i-1, V_i, V_i+1, V_i+2, V_i+3) \right|

It also seems odd to take a median of 6 values. Wouldn't an odd number centered on the index *i* that the QC refers to make more sense?

**Review by callumrollo — CHANGES_REQUESTED** [review]

File: src/tests.adoc (hunk 1 above)
Comment (callumrollo) [reply src/tests.adoc:32]: The spike test defined in the cited document [BGC-Argo quality control manual for nitrate concentration](https://archimer.ifremer.fr/doc/00732/84370/89380.pdf) defines a median filter with 5 values, centered on the value being tested.

File: src/realtime_qc.bib [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+@article{johnson2021bgc,
+  title={BGC-Argo quality control manual for nitrate concentration},
+  author={Johnson, Kenneth and Maurer, Tanya and Plant, Joshua and Bittig, Henry and Schallenberg, Christina and Schmechtig, Catherine},
+  year={2021}
```
Comment (callumrollo) [inline src/realtime_qc.bib:5]: I recommend we add the doi for this resource: 10.13155/84370

File: src/tests.adoc (hunk 1 above)
Comment (castelao) [reply src/tests.adoc:32]: @callumrollo, thanks! My bad.

File: src/realtime_qc.bib (hunk 2 above)
Comment (castelao) [reply src/realtime_qc.bib]: Yes, good idea!

**Review by callumrollo — APPROVED** [review]

**Comment by castelao** [comment]
This PR was initiated in early stage, before the review rules were defined. It's easier to merge this one and any required adjustment or fix to be addressed in future PRs following the editing rules.

**Pull request merged**